torch/_subclasses/functional_tensor.py

```diff
@@ -97,6 +97,15 @@ class FunctionalTensor(torch.Tensor):
         torch.ops.aten._batch_norm_impl_index.default,  # type: ignore[has-type]
         torch.ops.aten.cudnn_batch_norm.default,  # type: ignore[has-type]
         torch.ops.aten.miopen_batch_norm.default,  # type: ignore[has-type]
+        torch.ops.aten.atleast_1d.default,  # type: ignore[has-type]
+        torch.ops.aten.atleast_2d.default,  # type: ignore[has-type]
+        torch.ops.aten.atleast_3d.default,  # type: ignore[has-type]
+        torch.ops.aten.cartesian_prod.default,  # type: ignore[has-type]
+        torch.ops.aten.conj_physical.default,  # type: ignore[has-type]
+        torch.ops.aten.alpha_dropout.default,  # type: ignore[has-type]
+        torch.ops.aten.feature_dropout.default,  # type: ignore[has-type]
+        torch.ops.aten.feature_alpha_dropout.default,  # type: ignore[has-type]
```

Comment:
@tugsbayasgalan what's the plan with all of these? It seems like a silent incorrectness bug if an operator misrepresents its schema (unless it is one of the unsafe_* ops). If we agree that these are bugs, can you file some issues for them?

Reply:
I think for any of these "maybe-aliasing/mutation ops", our options are something like:

(1) always decompose them, force users of export to live with it

(2) for maybe-aliasing ops: change the op to never alias via CoW (only for export, not eager)

(3) for maybe-aliasing ops: change the op to never alias for both export and eager

Right now...

*  export is doing (1) for all of these ops

* except for dropout export does (2) (and it carefully tries to error on programs whose semantics diverge due to the dropout behavior change from eager)

* and in theory we might do (3) for `reshape()`, if we are willing to sit through the warn / deprecation cycle? I'm not sure what the progress here has been though.